buildpack.toml

```diff
@@ -18,8 +18,6 @@ id = "heroku-20"
 id = "heroku-22"
 
 [metadata]
-
 [metadata.release]
-
-[metadata.release.image]
+[metadata.release.docker]
```

Comment:
Sorry I missed this on previous PRs. I don't love `docker` here. Docker is a trademark and a company name. If we were to host this image in a different container registry (like say AWS ECR), `docker` wouldn't make sense here. I'd rather stick with brand agnostic naming here. The artifact we're pointing at is an OCI image. `image`, `registry`, or `url`, `repository` are all better IMO.

Reply:
filed as an issue - https://github.com/heroku/languages-github-actions/issues/45